modules/animation/include/modules/animation/animationcontroller.h

```diff
@@ -89,9 +89,6 @@ class IVW_MODULE_ANIMATION_API AnimationController : public AnimationControllerO
     /// Advances the animation to the next time step in playing state.
     void tick();
 
-    /// Advances the animation to the next time step in rendering state.
-    void tickRender();
```

Comment:
I don't think that the changes to animation belongs in this PR...

Reply:
Well the PR is a draft...